Make image description responsive in lightbox

ihsavru opened the pull request:
Fixes #7789
Width>786px

![screenshot from 2017-12-17 12 52 38](https://user-images.githubusercontent.com/22816171/34077428-e5b59056-e329-11e7-92da-86b0f279e686.png)

Width<786px
![screenshot from 2017-12-17 12 47 53](https://user-images.githubusercontent.com/22816171/34077381-d84c6d82-e328-11e7-9f5e-7d0c6d924e5f.png)
Width<511px
![screenshot from 2017-12-17 12 47 37](https://user-images.githubusercontent.com/22816171/34077382-de6225f4-e328-11e7-8521-feabd2c4ad02.png)

rishig commented:
This looks great, thanks @ihsavru! Sending to @brockwhittaker for final review.

brockwhittaker commented:
There's a couple things off with this:

1. The screen is no longer vertically centered now (providing a larger gap on the bottom.
<img width="1366" alt="screen shot 2017-12-29 at 4 54 32 pm" src="https://user-images.githubusercontent.com/10321399/34450240-fe753770-ecb8-11e7-8fbe-7f82a3e9dee8.png">

This is because you changed `- 65px` => `- 95px`. Not sure why.

2. See if the `font-size: 1em` is necessary. I don't *think* it is, so you may want to just remove the declaration.

ihsavru commented:
@brockwhittaker  I changed the height because the images at the bottom were getting cut off  
![screenshot from 2017-12-30 17 46 27](https://user-images.githubusercontent.com/22816171/34454146-e7a6c18a-ed8a-11e7-9c68-1ebeecf262ec.png)

I changed it back to 65px for screen size greater than 736px
![screenshot from 2017-12-30 17-54-11](https://user-images.githubusercontent.com/22816171/34454165-7182b76a-ed8b-11e7-86ff-9a4e6453c416.png)
And to 101px for less than that
![screenshot from 2017-12-30 18-03-00](https://user-images.githubusercontent.com/22816171/34454174-bce6333a-ed8b-11e7-9e0e-01ae28cf828e.png)

rishig commented:
Merged as #8119. Thanks again for your work on this!
